app/namespaces/data_provider/test_data_provider_endpoints.py

```diff
@@ -0,0 +1,107 @@
+from app import db as _db
+from app.serotracker_sqlalchemy.models import DashboardSource
+
+
+def test_healthcheck(client):
+    response = client.get('/healthcheck/')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is str
+    assert data == 'The healthcheck endpoint was hit.'
+
+
+# Test get filter options
+def test_get_filter_options(client):
+    response = client.get('/data_provider/filter_options')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is dict
+
+
+# Test get records details
+def test_get_record_details(client):
+    source_id = str(_db.session.query(DashboardSource.source_id).all()[0][0])
+    response = client.get(f'/data_provider/record_details/{source_id}')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is dict
```

Comment:
do we wanna do any more granular asserts where we check that the source_id is actually contained as a key in the result (i.e. so we confirm it got the correct record corresponding to the desired source id)?

Reply:
whoops missed that

Reply:
I don't think this endpoint actually returns source_id currently